ruby-challenges.rb

```diff
@@ -32,3 +69,27 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+
+def palindrome?(string)
+  normalized_string = string.downcase
+  reversed_string = normalized_string.reverse
```

Comment:
Nice logic, though you can throw both of these in your conditional without storing them in variables, saving you some lines of code